django/otto/views.py

```diff
@@ -654,3 +662,25 @@ def cost_dashboard(request):
         "cost_type_options": cost_type_options,
     }
     return render(request, "cost_dashboard.html", context)
+
+
+def user_cost(request):
+    today_cost = cad_cost(Cost.objects.get_user_cost_today(request.user))
+    weekly_max = request.user.this_week_max
+    this_week_cost = cad_cost(Cost.objects.get_user_cost_this_week(request.user))
+    cost_percent = max(
+        min(int(100 * this_week_cost / weekly_max if weekly_max else 0), 100), 1
+    )
+    cost_tooltip = (
+        f"${this_week_cost:.2f} / ${weekly_max:.2f} {_('this week')}<br>"
```

Comment:
Small thing but in French, the $ sign goes after the number. Not sure how this is handled in python. In javascscript I usually stick to Intl.NumberFormat